bootstrap/terraform/values/argocd.yaml

```diff
@@ -110,55 +110,40 @@ configs:
           end
 
           return health_status
-      %{ if crossplane_upjet_aws_provider_enable }
-      "*.aws.upbound.io/*":
+
+      "*.upbound.io/*":
```

Comment:
There is already "*.upbound.io/*" higher up in this file.

Reply:
we have one for *.upbound.io/*, another one for awsblueprints.io/* and another for *.crossplane.io/*.